tests/e2e/features/artifacts/mainPage/menu.feature

```diff
@@ -0,0 +1,15 @@
+@menuItems @smoke
+Feature: Menu
+
+  Background:
+    Given Connect Metamask extension with login action
+
+  @id1484
+  Scenario: Check Logout
+    Given I am on the Main page
+    When I click by "partial class" with "account-name" value
+    When I click by text " Logout "
```

Comment:
Is it possible to use more general way to get this element without spaces? can we catch it by `//*[@class='menu-options']//button[contains(text(), 'Logout')] `? The potential problem is what if we will have more or less than 1 space before\after this word?

Reply:
@pcheremu requested changes made - could you please review again?